stock_picking_batch_planner/models/stock_picking_type.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import api, fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    plan_origin = fields.Boolean(
+        default=False, help="Allow planning batches/waves in origin moves"
+    )
```

Comment:
The text implies that you can plan Waves with the origin movements from Batches of this type of operation.
Based on the current behavior, it should be something like "Planned Waves" and indicate that the criteria for this type of operation will be used when planning Waves for this type of operation.

```suggestion
    plan_wave = fields.Boolean(
        string="Planned Waves",
        default=False, 
        help="Place Stock Operations in waves when planned from the Batch of the destiny move.\n"
                 "Use 'Wave Grouping' criteria to decide how to plan waves",
    )
```